src/arkode/arkode_impl.h

```diff
@@ -189,6 +189,43 @@ typedef int (*ARKTimestepGetGammasFn)(void* arkode_mem, sunrealtype* gamma,
 typedef int (*ARKTimestepFullRHSFn)(void* arkode_mem, sunrealtype t, N_Vector y,
                                     N_Vector f, int mode);
 typedef int (*ARKTimestepStepFn)(void* arkode_mem, sunrealtype* dsm, int* nflag);
+typedef int (*ARKTimetepSetUserDataFn)(void* arkode_mem, void* user_data);
+typedef int (*ARKTimestepPrintAllStats)(void* arkode_mem, FILE* outfile,
+                                        SUNOutputFormat fmt);
+typedef int (*ARKTimestepWriteParameters)(void* arkode_mem, FILE* fp);
+typedef int (*ARKTimestepResize)(void* arkode_mem, N_Vector ynew,
+                                 sunrealtype hscale, sunrealtype t0,
+                                 ARKVecResizeFn resize, void* resize_data);
+typedef int (*ARKTimestepReset)(void* arkode_mem, sunrealtype tR, N_Vector yR);
+typedef void (*ARKTimestepFree)(void** arkode_mem);
```

Comment:
Maybe just a `void *` since it's not freeing the arkode_mem itself, just properties within.